requirements-dev.txt

```diff
@@ -1,8 +1,8 @@
 black==22.6.0
 flake8==5.0.4
-mypy==0.971
+mypy==1.8
 pytest>=7.2.0
 sphinx-autodoc-typehints[type-comment]>=1.19.3
 sphinx-rtd-theme
 rb>=1.9.0
-redis-py-cluster>=2.1.0
+redis>=5.0.1
```

Comment:
Should I remove Redis version, and keep it as a requirement for installation, or just include it with the package? 

I'm +1 on shipping this with Redis. lmk wyt